### ISSUE TITLE
Cargo build fails

### ISSUE THREAD

**Issue by kcalvinalvin** [rust-bitcoin/murmel#46]
Just git cloning the master branch and ```cargo build``` is failing.

log:

```
[I] > cargo build
   Compiling hammersbald v2.4.0
error[E0603]: module `export` is private
   --> /home/calvin/.cargo/registry/src/github.com-1ecc6299db9ec823/hammersbald-2.4.0/src/bitcoin_adaptor.rs:34:12
    |
34  | use serde::export::PhantomData;
    |            ^^^^^^ private module
    |
note: the module `export` is defined here
   --> /home/calvin/.cargo/registry/src/github.com-1ecc6299db9ec823/serde-1.0.123/src/lib.rs:275:5
    |
275 | use self::__private as export;
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0603`.
error: could not compile `hammersbald`

To learn more, run the command again with --verbose.
```

**Comment by sgeisler** [comment]
Murmel isn't that well maintained. The error you see is clearly a mishap in `hammersbald` (probably by IntelliJ Rust, which can auto-import stuff and accidentally took `PhantomData` from `serde::export` instead of `std::markers`). I guess a recent serde update fixed the std re-export and broke the build that way.

Version 3 of `hammersbald` already has a fix for it ([relevant commit](https://github.com/rust-bitcoin/hammersbald/commit/267a0306d97738e7a6b2f3d2600451c015b583ee#diff-860b5cb8a27a9fe7603ec04f5ac57b3eb957cf421c606cb7e6b6a59bb42533b8L34)). You could try to migrate murmel to the new `hammersbald` version or just fix the import and use your patched version locally.

**Comment by stevenroose** [comment]
https://github.com/rust-bitcoin/murmel/pull/47 fixes this issue, @kcalvinalvin could you verify that it works for you?

**Comment by kcalvinalvin** [comment]
> #47 fixes this issue, @kcalvinalvin could you verify that it works for you?

Yes it works for me on commit hash `8b93a33884956970c8c87bde2d00c934d8bca440`

**Comment by chrisguida** [comment]
Can confirm that master was broken as described above, and https://github.com/rust-bitcoin/murmel/pull/47 fixes it
